src/prog_models/prognostics_model.py

```diff
@@ -1075,9 +1075,22 @@ def output(x):
     def __sizeof__(self):
         return getsizeof(self)
 
-    def calc_error(self, times: List[float], inputs: List[InputContainer], outputs: List[OutputContainer], **kwargs) -> float:
-        """Calculate Mean Squared Error (MSE) between simulated and observed
+    def check_iterable(self, iter, name, loc = False):
```

Comment:
This isn't something specific to PrognosticsModel. Could we move this to be its own function in utils.